http4s/client/js/src/test/scala/endpoints4s/http4s/client/Http4sClientEndpointsJsonSchemaTest.scala

```diff
@@ -48,7 +50,7 @@ class Http4sClientEndpointsJsonSchemaTest
       port = Some(stubServerPort)
     ),
     Uri.Scheme.http,
-    FetchClientBuilder[IO].create
+    FetchClientBuilder[IO].withRequestTimeout(FiniteDuration(2, TimeUnit.SECONDS)).create
```

Comment:
Why is this change necessary?

Reply:
Oh, I see, there is no need to introduce the timeout configuration in the Http4s interpreter trait (like you did with the fetch interpreter, for instance), because the Http4s client can already be configured with a timeout by the users. That’s great!

I wonder if we should follow the same approach with the sttp interpreter then? Let the users configure the timeout in the underlying backend? Would that be possible?

Reply:
I extracted the config from the mixin in sttp tests, is that what you meant?

Reply:
Hey Romain ! No, I am sorry, I wasn't clear. What I meant is that maybe instead of introducing a timeout setting in the sttp interpreter it would be possible to do nothing about the timeout in endpoints4s and let sttp handle it itself. Just like you did here with this http4s interpreter: you didn't change anything, but you simply configured the timeout in http4s itself.

Reply:
Okay I understand. I think this was the first thing I tried.

I just tried it again, using this config:

```scala
val backend = TryHttpURLConnectionBackend(options = SttpBackendOptions.connectionTimeout(FiniteDuration(2, TimeUnit.SECONDS)))
```

but unfortunately, the interpreter doesn't timeout

Reply:
I think that the global config it made for the connection timeout (at tcp level) and the other for the read.

It is not the same timeout:
https://sttp.softwaremill.com/en/latest/conf/timeouts.html

Reply:
Okay, I think I've actually found how to it.

I just committed it.